app/controllers/facilities_controller.rb

```diff
@@ -254,4 +255,8 @@ def set_column_class
     @columns = "columns" if SettingsHelper.feature_on?(:product_list_columns)
   end
 
+  def set_quicklinks
```

Comment:
If you don't want to inline the `feature_on?` in the view, I think the view would read cleaner if this were:
```ruby
def quicklinks?
  SettingsHelper.feature_on?(:product_quicklinks)
end
helper_method :quicklinks?
```

Then you can do `if quicklinks?` inside the view.